animation_nodes/nodes/sound/MIDI_Track_Info.py

```diff
@@ -0,0 +1,15 @@
+import bpy
+from ... base_types import AnimationNode, ListTypeSelectorSocket
+
+class MidiTrackNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_MidiTrackInfoNode"
+    bl_label = "MIDI Track Info"
+    bl_width_default = 180
+
+    def create(self):
+        self.newInput("MIDITrack", "Track", "track")
+        self.newOutput("MIDINoteList", "Notes", "notes")
+
+    def execute(self, track):
+        if track is None: return None
```

Comment:
Might as well return the track index and track name.

Reply:
You mean add two new output with these values ?

Reply:
Yes.